README.md

```diff
@@ -92,14 +92,21 @@ Your `~/.vimrc.local` might look like this:
 If you don't wish to install a vim plugin from the default set of vim plugins in
 `.vimrc.bundles`, you can ignore the plugin by calling it out with `UnPlug` in
 your `~/.vimrc.bundles.local`.
-blacklist in `~/.vimrc.ignorebundles`:
 
     " Don't install vim-scripts/tComment
     UnPlug 'tComment'
 
 `UnPlug` can be used to install your own fork of a plugin or to install a shared
 plugin with different custom options.
 
+    " Only load vim-coffee-script if a Coffeescript buffer is created
+    UnPlug 'vim-coffee-script'
+    Plug 'kchmck/vim-coffee-script', { 'for': 'coffee' }
```

Comment:
Update: I added lazy loading to the non-rails.vim plugins that seemed like the most likely candidates for load time improvement (only used for specific filetypes, didn't need to be preloaded like rails.vim): `vim-coffee-script`, `vim-go`, `vim-ruby`, and `vim-rake`, `vim-slim`. I wasn't able to shave any noticeable startup time when starting Neovim or Vim by lazy loading. 😞